.github/workflows/publish-napi.yml

```diff
@@ -0,0 +1,161 @@
+---
+name: Publish @bitwarden/sdk-napi
+run-name: Publish @bitwarden/sdk-napi ${{ inputs.release_type }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: "Release Options"
+        required: true
+        default: "Release"
+        type: choice
+        options:
+          - Release
+          - Dry Run
+      npm_publish:
+        description: "Publish to NPM registry"
+        required: true
+        default: true
+        type: boolean
```

Comment:
```suggestion
```

Since this workflow only publishes to NPM, we can remove the input.  It would do nothing if the box was not checked.

Reply:
Fixed.